### DISCUSSION TITLE
Running LIS with NoahMP LSM and ERA5 metforcing

### DISCUSSION THREAD

**Discussion by alexamarco** [NASA-LIS/LISF#1579] [Q&A] [answered]
Hello All,

I am trying to run LIS using NoahMP-4.0.1 as the LSM and ERA5 meteorological forcing.  Unfortunately, I'm running into an error with reading the ERA5 data. Below are some examples of the errors I'm getting:

x1001c1s6b0n1.hsn.ex4000.erdc.hpc.mil: rank 38 died from signal 9
forrtl: error (78): process killed (SIGTERM)
Image              PC                Routine            Line        Source      
libpthread-2.31.s  00001493EA0E1910  Unknown               Unknown  Unknown
libimf.so          00001493ED2DC5DD  __isgreater           Unknown  Unknown
libnetcdf.so.18.0  00001493ED1917DA  nc4_convert_type      Unknown  Unknown
libnetcdf.so.18.0  00001493ED18AC6C  NC4_get_vars          Unknown  Unknown
libnetcdf.so.18.0  00001493ED18A363  NC4_get_vara          Unknown  Unknown
libnetcdf.so.18.0  00001493ED12371E  nc_get_vara_float     Unknown  Unknown
libnetcdff.so.7.0  00001493E9B798EB  nf_get_vara_real_     Unknown  Unknown
libnetcdff.so.7.0  00001493E9BE3735  netcdf_mp_nf90_ge     Unknown  Unknown
LIS.7.3_ERA5.exe   000000000138C8B5  read_era5_                323  read_era5.F90
LIS.7.3_ERA5.exe   0000000000CF16F3  get_era5_                 184  get_era5.F90
LIS.7.3_ERA5.exe   000000000070ECCB  lis_metforcingmod        1090  LIS_metforcingMod.F90
LIS.7.3_ERA5.exe   0000000001437DF1  retrospective_run         194  retrospective_runMod.F90
LIS.7.3_ERA5.exe   0000000000E132A7  MAIN__                     70  lisdrv.F90
LIS.7.3_ERA5.exe   0000000000414CDD  Unknown               Unknown  Unknown
libc-2.31.so       00001493E88F224D  __libc_start_main     Unknown  Unknown
LIS.7.3_ERA5.exe   0000000000414C0A  Unknown               Unknown  Unknown
forrtl: error (78): process killed (SIGTERM)
Image              PC                Routine            Line        Source      
libpthread-2.31.s  0000148E74A76910  Unknown               Unknown  Unknown
libimf.so          0000148E742DC5DD  __isgreater           Unknown  Unknown
libnetcdf.so.18.0  0000148E741917DA  nc4_convert_type      Unknown  Unknown
libnetcdf.so.18.0  0000148E7418AC6C  NC4_get_vars          Unknown  Unknown
libnetcdf.so.18.0  0000148E7418A363  NC4_get_vara          Unknown  Unknown
libnetcdf.so.18.0  0000148E7412371E  nc_get_vara_float     Unknown  Unknown
libnetcdff.so.7.0  0000148E70B798EB  nf_get_vara_real_     Unknown  Unknown
libnetcdff.so.7.0  0000148E70BE3735  netcdf_mp_nf90_ge     Unknown  Unknown
LIS.7.3_ERA5.exe   000000000138C8B5  read_era5_                323  read_era5.F90
LIS.7.3_ERA5.exe   0000000000CF16F3  get_era5_                 184  get_era5.F90
LIS.7.3_ERA5.exe   000000000070ECCB  lis_metforcingmod        1090  LIS_metforcingMod.F90
LIS.7.3_ERA5.exe   0000000001437DF1  retrospective_run         194  retrospective_runMod.F90
LIS.7.3_ERA5.exe   0000000000E132A7  MAIN__                     70  lisdrv.F90
LIS.7.3_ERA5.exe   0000000000414CDD  Unknown               Unknown  Unknown
libc-2.31.so       0000148E6F8F224D  __libc_start_main     Unknown  Unknown
LIS.7.3_ERA5.exe   0000000000414C0A  Unknown               Unknown  Unknown

I looked through my forcing data and it has all of the required variables needed for LIS.  Has anyone tried to use ERA5 metforcing with NoahMP before and run into this issue?

Thank you for reading through this discussion.  Any advice is much appreciated!

**Comment by dmocko** [comment, marked as answer]
Hi @alexamarco 

Can you please look at the end of your lislog files?  Not only lislog.0000, but please see if there is any warning or error message
at the end of any of them.

I can't tell just from what you have shared above if this is an issue we have seen before or not with the ERA5 metforcing.

That said, one known issue is that the ERA5 forcing dataset that we have shared is only available over land points.
The land points are defined by the ERA5 dataset, and they may not match the LANDMASK that you are using.

If you are trying to run at a land point where ERA5 forcing is not available, you should get a error message in your lislog.

I believe you are running on Carpenter, so if you would like to send me an e-mail pointing to your directory, I can take a look
at your lis.config, lislogs, and other files.  Please be sure that I have permissions to look at them, if you want me to see them.

**Reply by alexamarco** [reply]
Hi David,

Thank you for your help!  I just sent you an email.
